src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj

```diff
@@ -121,6 +121,21 @@
     <NativeRuntimeAsset Include="$(_diaSymReaderPathIfExists)" />
   </ItemGroup>
 
+  <!-- VS uses this file to show the target framework in the drop down. -->
+  <Target Name="CreateDotVersionFile" DependsOnTargets="InitializeSourceControlInformationFromSourceControlManager" BeforeTargets="GetFilesToPublish">
```

Comment:
Please condition this target on `$(DisableSourceLink) != true` similar to what we do for the native version file: https://github.com/dotnet/arcade/blob/2b430e5bbfaec37a6cead2f0cf79157f01f1a623/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.targets#L47.

Reply:
```suggestion
  <Target Name="CreateDotVersionFile"
          DependsOnTargets="InitializeSourceControlInformationFromSourceControlManager"
          BeforeTargets="GetFilesToPublish"
          Condition="'$(DisableSourceLink)' != 'true'">
```